code/modules/antagonists/traitor/objectives/infect.dm

```diff
@@ -136,34 +138,35 @@
 		//don't take an objective target of someone who is already obliterated
 		fail_objective()
 
-
 /datum/traitor_objective/target_player/infect/proc/on_target_death()
 	SIGNAL_HANDLER
 	if(objective_state == OBJECTIVE_STATE_INACTIVE)
 		//don't take an objective target of someone who is already dead
 		fail_objective()
 
-
 /obj/item/reagent_containers/hypospray/medipen/manifoldinjector
 	name = "EHMS autoinjector"
 	desc = "Experimental Hereditary Manifold Sickness autoinjector."
 	icon_state = "tbpen"
 	inhand_icon_state = "tbpen"
 	base_icon_state = "tbpen"
-	var/used = 0
+	//Was the injector used on someone yet?
+	var/used = FALSE
```

Comment:
needs to be below existing variables

Reply:
Ohhhh, put the new variable below. Thanks!